python/lsst/pipe/tasks/multiBand.py

```diff
@@ -604,11 +642,23 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
 
         # Set psfcache
         # move this to run after gen2 deprecation
-        exposure = inputs.pop("exposure")
-        exposure.getPsf().setCacheCapacity(self.config.psfCache)
+        if self.config.useCellCoadds:
+            multiple_cell_coadd = inputs.pop("exposure_cells")
+            stitched_coadd = multiple_cell_coadd.stitch()
+            exposure = stitched_coadd.asExposure()
+            background = inputs.pop("background")
+            exposure.image -= background.getImage()
+
+            ccdInputs = stitched_coadd.ccds
+            apCorrMap = stitched_coadd.ap_corr_map
+            band = inputRefs.exposure_cells.dataId["band"]
```

Comment:
This block is common enough to merit being put into a function somewhere, even if the names of the connections need to be parameters.